site/templates/common/cta_buttons.jet

```diff
@@ -34,9 +42,11 @@
       {{end}}
 
       {{if showCanBeWatchedButton}}
-        {{dataURL := .CustomFields.GetString("can_be_watched_button_link", "")}}
-        {{dataLabel := .CustomFields.GetString("can_be_watched_button_text", "")}}
-        <can-be-watched-button data-slug="{{slug}}" data-url="{{dataURL}}" data-label="{{dataLabel}}"></can-be-watched-button>
+        <can-be-watched-button data-slug="{{slug}}" data-url="{{canBeWatchedButtonLink}}" data-label="{{canBeWatchedButtonText | i18n}}"></can-be-watched-button>
+      {{end}}
+
+      {{if showExternalPurchaseButton}}
+        <external-purchase-button data-slug="{{slug}}" data-url="{{externalPurchaseButtonLink}}" data-label="{{externalPurchaseButtonText | i18n}}"></external-purchase-button>
```

Comment:
Daaaamn it never occurred to me that i18n could be piped!!!!!  That's so much cleaner!